### PR TITLE
fix: #941 Updating formatting of New Thread > Create a Poll panel

### PR DESCRIPTION
**Pull request by ruesato**
Now sized properly and allows scrolling to view all controls
Editor panel also allows space for other panels (top, post contents)

### CODE REVIEW

File: modules/ept-polls/client/poll_creator.html [hunk 1]
```diff
@@ -2,11 +2,11 @@
   <div class="section-poll-questions">
     <span class="section-header">Create a Poll</span>
     <!-- Poll Question -->
-    <h1 class="poll-question">Question:</h1>
+    <label for="question" class="poll-question">Question:</label>
     <input type="text" id="question" ng-model="poll.question" maxlength="255">
   
     <!-- Poll Answers -->
-    <h1 class="poll-answers">Answers:</h1>
+    <label for="answer-1" class="poll-answers">Answers:</h1>
```
Comment (unenglishable) [inline modules/ept-polls/client/poll_creator.html]: replace `</h1>` with `</label>`

File: modules/ept-posts/client/directives/editor.html [hunk 2]
```diff
@@ -44,7 +44,7 @@
           </div>
         </div>
 
-        <poll-creator poll="thread.poll" valid="thread.pollValid" ng-if="thread.addPoll"></poll-creator>
+
```
Comment (unenglishable) [inline modules/ept-posts/client/directives/editor.html]: remove line

**Review by unenglishable — APPROVED** [review]
fixes in

**Review by akinsey — APPROVED** [review]
Tested and appears to be working fine!

**Pull request merged**